bin/dev

```diff
@@ -97,7 +97,8 @@ runPrecommit(){
 
 # One time setup to add pre-commit hook
 runSetup(){
-    cp bin/pre-commit .git/hooks/pre-commit
+    rm .git/hooks/pre-commit
```

Comment:
it might be good to not have this line and us devs remove it manually (or check if `pre-commit` exists), otherwise it will error out.

Reply:
Good point.